client/my-sites/plugins/index.web.js

```diff
@@ -89,8 +90,11 @@ export default function ( router ) {
 		clientRender
 	);
 
+	router( `/${ langParam }/plugins/plans`, notFound, makeLayout );
+
 	router(
-		'/plugins/plans/:site',
+		`/${ langParam }/plugins/plans/:site`,
+		redirectWithoutLocaleParamIfLoggedIn,
```

Comment:
What issue does including locale when logged in cause? Is that because a logged-in user already has a locale?

Reply:
Yes, I believe a user's locale takes precedence over the location in the URL, so when the user is logged in, the location is stripped from the URL. I think just so it's not confusing if the languages shouldn't match up for some reason.